src/components/trends/common/Tooltip.css

```diff
@@ -0,0 +1,40 @@
+.tooltip {
+  display: block;
+  font-size: 14px;
```

Comment:
As mentioned in our meeting earlier, pull in updates from master and use the `defaultSize` from typography.css plz.